common/libs/nodes/rpc/index.js

```diff
@@ -71,13 +63,19 @@ export default class RpcNode extends BaseNode {
 
   async generateTransaction(
```

Comment:
I'll just make it a function in `libs/transaction.js` for now that takes in a node, but leave a note saying we might want to figure out a better way. I'm not sure I see the bigger picture of transactions in MEW beyond Send yet, and I don't want to make a large architectural decision on incomplete information.